Plugin/CppApi/source/CoordinateConversionController.cpp

```diff
@@ -411,17 +380,15 @@ void CoordinateConversionController::clearOptions()
 }
 
 /*!
-  \brief Gets the name of this tool.
+  \brief Returns the name of this tool.
```

Comment:
we didn't override this for the compass tool... I'm not sure whether we can just rely on the base doc for these methods?

Reply:
> we didn't override this for the compass tool... 

Why not?

Reply:
i don't know - it's in this PR I think?